list

```diff
@@ -633,6 +645,7 @@ itsh.bo
 itvty.com
 ity.im
 ix.sk
+izilink.ru
```

Comment:
Any working example for it?

Reply:
Original link (https://izilink.ru/xCCRo) still works for me, instant 301 redirect.
Archive: https://archive.is/V3ZnN